packages/client/src/services/KayentaApiService.ts

```diff
@@ -12,7 +12,7 @@ import {
 
 const kayentaClient = axios.create({
   baseURL: `${window.location.origin}/kayenta/`,
-  timeout: 5000
+  timeout: 10000
```

Comment:
Was stuff timing out at 5 seconds?

Reply:
Yes, on certain reloads I was getting the error "Failed to fetch standalone canary analysis pipeline execution status from Kayenta". When I increased the timeout, the page loaded without error.